werkzeug/serving.py

```diff
@@ -62,10 +66,16 @@ def _get_openssl_crypto_module():
 
 
 try:
-    from SocketServer import ThreadingMixIn, ForkingMixIn
+    if can_fork:
+        from SocketServer import ThreadingMixIn, ForkingMixIn
+    else:
+        from SocketServer import ThreadingMixIn
     from BaseHTTPServer import HTTPServer, BaseHTTPRequestHandler
 except ImportError:
-    from socketserver import ThreadingMixIn, ForkingMixIn
+    if can_fork:
+        from socketserver import ThreadingMixIn, ForkingMixIn
+    else:
+        from socketserver import ThreadingMixIn
```

Comment:
Is there a more elegant way of doing this than having an `if`/`else` in both the `try` and `except` statements and declaring a stub `ForkingMixIn` class in both?